src/native/windows/apps/prunsrv/prunsrv.c

```diff
@@ -2204,8 +2217,11 @@ void __cdecl main(int argc, char **argv)
     gStdwrap.szLogPath = SO_LOGPATH;
     /* Only redirect when running as a service */
     if (lpCmdline->dwCmdIndex == 2) {
+        apxLogWrite(APXLOG_MARK_DEBUG "redirect!");
         gStdwrap.szStdOutFilename = SO_STDOUTPUT;
         gStdwrap.szStdErrFilename = SO_STDERROR;
+    } else {
+        apxLogWrite(APXLOG_MARK_DEBUG "NO redirect!");
```

Comment:
Same comments as above. Also "NO" doesn't need to be capitalized. Sentences start with a capital, sure.

Reply:
I have removed that one too.